### PR TITLE
feat(playground): Added support for MySQL TCTC-3401

### PR DESCRIPTION
**Pull request by lukapeschke**
depends-on #1542

### CODE REVIEW

**Review by Sanix-Darker — COMMENTED** [review]
LGTM, a small rebase looks needed !

File: docker-compose.yml [hunk 1]
```diff
@@ -49,6 +65,11 @@ services:
 
       GOOGLE_BIG_QUERY_CREDENTIALS_FILE: "/secrets/bigquery-credentials.json"
 
+      MYSQL_HOST: mysql
+      MYSQL_PASSWORD: password
+      MYSQL_USER: dbuser
+      MYSQL_DATABASE: playground_db
```
Comment (Sanix-Darker) [inline docker-compose.yml:71]:
```suggestion
      MYSQL_HOST: "mysql"
      MYSQL_PASSWORD: "password"
      MYSQL_USER: "dbuser"
      MYSQL_DATABASE: "playground_db"
```

consistency ?

File: playground/init-mysql/default-dataset.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  `Value3` INTEGER,
+  `Value4` INTEGER,
+  `Groups` VARCHAR(255)
+);
+
+LOAD DATA LOCAL INFILE '/var/lib/mysql-files/default-dataset.csv'
+INTO TABLE default_dataset
+FIELDS TERMINATED BY ','
+LINES TERMINATED BY '\n'
+IGNORE 1 ROWS (`Label`,`Value1`,`Value2`,`Value3`,`Value4`,`Groups`);
```
Comment (Sanix-Darker) [inline playground/init-mysql/default-dataset.sql:14]: 👌🏽

File: server/playground.py [hunk 3]
```diff
@@ -699,6 +700,73 @@ async def hangle_bigquery_post_request():
...
+    return {
+        row["table_name"]: json.loads(row["column_names"])
+        for _, row in pd.read_sql(
+            """SELECT t.table_name "table_name", JSON_ARRAYAGG(c.column_name) "column_names" """
+            "FROM information_schema.tables t "
+            "INNER JOIN information_schema.columns c ON t.table_name = c.table_name "
+            "WHERE t.table_type = 'BASE TABLE' AND t.table_schema = 'playground_db' "
+            "GROUP BY t.table_name "
+            "ORDER BY t.table_name;",
+            _mysql_connection(),
```
Comment (Sanix-Darker) [inline server/playground.py:735]:
```suggestion
        for _, row in pd.read_sql(
            """SELECT t.table_name table_name, JSON_ARRAYAGG(c.column_name) column_names
            FROM information_schema.tables t 
            INNER JOIN information_schema.columns c ON t.table_name = c.table_name 
            WHERE t.table_type = 'BASE TABLE' AND t.table_schema = 'playground_db' 
            GROUP BY t.table_name 
            ORDER BY t.table_name;""",
            _mysql_connection(),
```
much clear ? WDYT ?

Comment (lukapeschke) [reply server/playground.py:735]: Don't like it, looks bad when printed. And I hate left-indented multiline strings :smile:

**Review by Sanix-Darker — APPROVED** [review]

**Pull request merged**